docs/changelog.md

```diff
@@ -1,6 +1,11 @@
 _See [here](./versioning-and-upgrading.md) for information about versioning and upgrading_
 
 # TBD
+### Changes
+* Imports datastore service and api service clients to interact with both services. 
```

Comment:
I'd add "....clients from `example-microservices` to interact...." to clarify where this is coming from